### ISSUE TITLE
Optimize performance 

### ISSUE THREAD

**Issue by nirvanatikku** [nirvanatikku/bittrex-enhanced#2]
Some users are noting that Chrome slows down a bit. Need to optimize DOM manipulation and also better handle ETH price polling (move to background.js and persist in storage?)

**Comment by Utone** [comment]
Yes I simply can not live without this extension anymore but a few tabs grind my Macbook Pro to a slow freeze... 

It even at times just plainly crashed my mac after I had opened Chrome with a few trade tabs where I left off the day before... maybe eight Bittrex tabs, 3 polo and tone or two trading view tabs...

Fans began to work at max capacity They always do when running Chrome, but even Firefox or Safari have this to some extent....

It shut down and restarted to the screen which i can only imagine you getting scolded in Mandarin Chinese... The equivalent of A BSOD (but worse)  had to reboot and quickly close tabs might even been in safe mode as restarted with chrome open again so went on a boot crash loop... how insane is that?


Other than that the pages start to take longer to load and also get this 'object failed to load' pop up message, once out of three times I open a new Bittrex tab.


Hope you van improve the loading time, thanks for making such a great add-on!
